script/cms/compare-json.js

```diff
@@ -182,24 +225,48 @@ const compareJson = (baseGraphQlObject, baseCmsExportObject) => {
     }
   });
 
+  const arraysToCompare = [];
+
   // Get array of differences. Exclude new properties because transformed
   // CMS objects may have additional properties
-  return deepDiff(graphQlObject, cmsExportObject, (diffPath, key) => {
+  const diffs = deepDiff(graphQlObject, cmsExportObject, (diffPath, key) => {
+    // Store arrays for comparison by alternate method.
+    const cmsObject = Object.byArray(cmsExportObject, diffPath);
+    const graphQLObject = Object.byArray(cmsExportObject, diffPath);
```

Comment:
Is the `get` function from `src/platform/utilities/data/get.js` suitable for this? It would return `undefined` (or some other default) for paths that don't exist on the object, but I think in those cases, we should return `false` for the equailty check anyway.

Reply:
🤦  I forgot about `get`! Thanks, I'll use that instead!!

Reply:
I ran into problems using src/platform/utilities/data/get.js because we're not in a module, so I used lodash get instead.